autogen/parser.py

```diff
@@ -44,7 +44,8 @@ def read_pari_desc():
         ...   'section': 'transcendental'}
         True
     """
-    with open(os.path.join(pari_share(), b'pari.desc')) as f:
+    pari_desc = os.path.join(pari_share(), b'pari.desc')
```

Comment:
This should still be fixed as well.  In fact as currently written, now `pari_share()` can return either `bytes` or `str` (in the Python 3 sense).  So it should pick one (you know what I think the answer should be but I won't argue about it right now as long as it's consistent).